addons/website/static/src/less/website.snippets.less

```diff
@@ -470,3 +470,23 @@ div.carousel {
 #oe_main_menu_navbar + #wrapwrap .o_header_affix {
     top: @odoo-navbar-height;
 }
+
+// Facebook Page
+
+.o_facebook_page {
+    position: relative;
+    .o_facebook_alert {
+        position: relative;
+        .o_add_facebook_page {
+            cursor: pointer;
+        }
+    }
+}
+
+.editor_enable .o_facebook_page:before {
+    content: '';
+    display: block;
+    height: 100%;
+    width: 100%;
+    position: absolute;
```

Comment:
I might be mistaken but this is probably better:
```less
.o-position-absolute(0, 0, 0, 0);
```
to replace `height`, `width`, `position`.
In any case, be sure to check https://github.com/odoo/internal/wiki/LESS-coding-guidelines